lib/Lab/Moose/Instrument/SCPIBlock.pm

```diff
@@ -90,7 +90,7 @@ sub block_to_array {
     }
 
     my @floats = unpack(
-        $precision eq 'single' ? 'f*' : 'd*',
+        $precision eq 'single' ? 'f>*' : 'd*',
```

Comment:
Is this safe for all instruments using this?  If not it probably needs to add a parameter for endianness instead of using this change.